test/dlc_tests/container_tests/bin/benchmark/run_tensorflow_maskrcnn_training_performance_hpu

```diff
@@ -15,6 +15,7 @@ cd ${HOME_DIR}/artifacts/gaudi-test-suite
 git checkout r${GIT_BRANCH}
 python3 -m pip install -r requirements.txt
 python3 -m pip install -r test/benchmark/tf-requirements.txt
+python3 -m pip install pycocotools --upgrade
```

Comment:
Why is this required? Will it make the test backward incompatible?

Reply:
Required because of a software update in the maskrcnn dependancies. This allows the test to use proper dependancies for the latest version

Reply:
What about the older synai versions? Will that work normally for them?